keps/sig-node/4815-dra-partitionable-devices/README.md

```diff
@@ -1651,6 +1863,73 @@ devices:
     - name: memory-slices-0-7
 ```
 
+### Using DRA for the multi-host use-case
+
+In order to allocate a 2x4 TPU slice using the ResourceSlice
+[shown above](#defining-multi-host-devices), a ResourceClaim like the
+following can be used:
+
+```yaml
+apiVersion: resource.k8s.io/v1beta1
+kind: ResourceClaim
+metadata:
+  name: tpu-consumer-resource-claim
+spec:
+  devices:
+    requests:
+    - name: tpu-request
+      deviceClassName: tpu.google.com
+      selectors:
+      - cel:
+          expression: "device.capacity['tpu.google.com'].tpus == '8'"
```

Comment:
this works, and probably best in the KEP. in practice though we may create device classes for these

Reply:
Yeah, that could be a good solution. I think a lot of our tests currently assumes a one-to-one mapping between driver and class (but I might be missing something), so we should make sure these kind of use-cases does work as expected.